src/lib/sidebar/scheme_data.ts

```diff
@@ -197,3 +204,112 @@ export function getArbitraryScheme(
 ): SchemeData {
   return Object.values(schemeCollection.schemes)[0];
 }
+
+export function getRectifiedPipelineSchemeWithSuggestedValues(
```

Comment:
Are we going with the hint approach? Why are we modifying `scheme` at all? Can the implementation of this just be:

1) In `PipelineschemeForm`, call a helper function that finds the max estimated year from features, or undefined if nothing's filled out. Display a bit of text near that field with the value.
2) Similar for summing cost per feature

I'm getting errors when I open "scheme details" with prior data; people are going to hit those tomorrow. We can fix these cases, but I'm finding code in here very confusing -- it's mutating the scheme and trying to fill out more default values. If we need to do that, centralized in `backfill` like we've been doing? But do we need to at all for hints -- shouldn't it just be a few readonly methods?